x-pack/plugins/stack_connectors/public/connector_types/slack_api/slack_api.tsx

```diff
@@ -52,4 +67,20 @@ export const getConnectorType = (): ConnectorTypeModel<
   },
   actionConnectorFields: lazy(() => import('./slack_connectors')),
   actionParamsFields: lazy(() => import('./slack_params')),
+  resetParamsOnConnectorChange: (
```

Comment:
maybe a better name would be `convertParamsBetweenGroups`?

Reply:
done